src/videojs.ads.js

```diff
@@ -3,98 +3,12 @@
  *
  * Common code to support ad integrations.
  */
-(function(window, document, videojs, undefined) {
-"use strict";
+(function(window, videojs, undefined) {
+'use strict';
 
 var
 
-  /**
-   * Copies properties from one or more objects onto an original.
-   */
-  extend = function(obj /*, arg1, arg2, ... */) {
-    var arg, i, k;
-    for (i=1; i<arguments.length; i++) {
-      arg = arguments[i];
-      for (k in arg) {
-        if (arg.hasOwnProperty(k)) {
-          obj[k] = arg[k];
-        }
-      }
-    }
-    return obj;
-  },
-
-  /**
-   * Add a handler for multiple listeners to an object that supports addEventListener() or on().
-   *
-   * @param {object} obj The object to which the handler will be assigned.
-   * @param {mixed} events A string, array of strings, or hash of string/callback pairs.
-   * @param {function} callback Invoked when specified events occur, if events param is not a hash.
-   *
-   * @return {object} obj The object passed in.
-   */
-  on = function(obj, events, handler) {
-
-    var
-
-      type = Object.prototype.toString.call(events),
-
-      register = function(obj, event, handler) {
-        if (obj.addEventListener) {
-          obj.addEventListener(event, handler);
-        } else if (obj.on) {
-          obj.on(event, handler);
-        } else if (obj.attachEvent) {
-          obj.attachEvent('on' + event, handler);
-        } else {
-          throw new Error('object has no mechanism for adding event listeners');
-        }
-      },
-
-      i,
-      ii;
-
-    switch (type) {
-      case '[object String]':
-        register(obj, events, handler);
-        break;
-      case '[object Array]':
-        for (i = 0, ii = events.length; i<ii; i++) {
-          register(obj, events[i], handler);
-        }
-        break;
-      case '[object Object]':
```

Comment:
were we ever even using this case in the code? This would be the only thing that possibly could break.